src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEMethodSymbol.cs

```diff
@@ -45,10 +45,12 @@ public SignatureData(SignatureHeader header, ImmutableArray<ParameterSymbol> par
         // This type is used to compact many different bits of information efficiently.
         private struct PackedFlags
         {
-            // We currently pack everything into a 32-bit int with the following layout:
+            // We currently pack everything into a 64-bit long with the following layout:
```

Comment:
>// We currently pack everything into a 64-bit long with the following layout: [](http://example.com/codeflow?start=12&length=77)

Do we really want to add another 4 bytes to every method symbol? Would it make sense to use an explicit layout for this struct and simply adding a byte field to hold the two new bits (since they are closely related)?